src/xcode/ENA/ENA/Source/Scenes/Settings/NotificationSettings/NotificationSettingsViewController.swift

```diff
@@ -23,26 +26,17 @@ class NotificationSettingsViewController: UIViewController, UITableViewDataSourc
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
-
-		tableView.delegate = self
-		tableView.dataSource = self
-		tableView.separatorColor = .enaColor(for: .hairline)
-
-		navigationItem.title = AppStrings.NotificationSettings.navigationBarTitle
-		navigationItem.largeTitleDisplayMode = .always
-		navigationController?.navigationBar.prefersLargeTitles = true
+		
+		setupView()
```

Comment:
Is it correct, that `setupView ` is called twice? See line 39.

Reply:
That is one too much. Removed one.